ios/docs/Controls/ListItem.md

```diff
@@ -0,0 +1,35 @@
+# ListItem
+
+## Overview
+`ListItem` can be used either in a `List` or standalone to represent a single item within a vertical container. It can display a title, optional subtitle, optional footer, optional leading content view, and optional trailing content view. You can also customize the top and bottom separators, accessory type, and action. When an `action` parameter is passed in, `ListItem` will render as a `Button`. When `action` is nil or no `action` parameter is passed in, `ListItem` will render as a `View`.
```

Comment:
Should `List` link to SwiftUI List documentation to make clear that there is no Fluent List control (at least yet)